zOSMF-Samples/workflow_sample_wizards_upgrade.xml

```diff
@@ -0,0 +1,327 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/********************************************************************************/
+/* Copyright 2019 IBM Corp.                                                     */
+/*                                                                              */
+/* Licensed under the Apache License, Version 2.0 (the "License");              */
+/* you may not use this file except in compliance with the License.             */
+/* You may obtain a copy of the License at                                      */
+/*                                                                              */
+/*    http://www.apache.org/licenses/LICENSE-2.0                                */
+/*                                                                              */
+/* Unless required by applicable law or agreed to in writing, software          */
+/* distributed under the License is distributed on an "AS IS" BASIS,            */
+/* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.     */
+/* See the License for the specific language governing permissions and          */
+/* limitations under the License.                                               */
+/********************************************************************************/
+-->
+ 
+<workflow xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                          xsi:noNamespaceSchemaLocation="../workflow/schemas/workflow_v1.xsd">    
+ 
+<!-- +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++ 
+	 This sample demonstrates the use of workflow upgrade feature to workflow customers, 
+	 as workflow author, they can decide multiple upgrade sections at the end of the XML 
+	 definition, choose whether cancel the old workflow or not after the upgrade complete
+	 by specifying the attribute "type" in their "version" element.
+	 
+	 Note, this sample of upgrade workflow is supposed to upgrade the workflow that created
```

Comment:
Since this is a copy of the workflow_sample_wizards, same my suggestions apply to this file as the other one.

Reply:
The same changed made for workflow_sample_wizards also made on workflow_sample_wizards_upgrade.